cmd/etcd.go

```diff
@@ -61,22 +61,22 @@ func newShowDDLInfoCommand() *cobra.Command {
 	return m
 }
 
-// newDelOwnerCampaign return a delete owner campaign subcommand of EtcdCommand
-func newDelOwnerCampaign() *cobra.Command {
+// newDelKeyCommand return a delete key subcommand of EtcdCommand
```

Comment:
s/return/returns
Add `.` at the end of a line.

Reply:
done.